scripts/bb-test-zfstests.sh

```diff
@@ -52,16 +52,9 @@ $ZFSTESTS $TEST_ZFSTESTS_OPTIONS \
     -d $TEST_ZFSTESTS_DIR \
     -s $TEST_ZFSTESTS_DISKSIZE \
     -r $TEST_ZFSTESTS_RUNFILE &
-CHILD=$!
-
-sleep 1
-TEST_LOG=$(ls -t /var/tmp/test_results/*/log | head -1)
-rm -f log
-ln -s $TEST_LOG log
-
-wait $CHILD
```

Comment:
Ah, my misunderstanding. I'll go fix test-runner real quick, then I can refresh this patch.